src/iinit.cpp

```diff
@@ -18,9 +17,11 @@
 #include <boost/lexical_cast.hpp>
 
 #include <nlohmann/json.hpp>
+#include <fmt/format.h>
```

Comment:
Well, since we're still debating stuff, can you flip the order of 19 and 20 for more alphabetization? :)